src/components/panel/panel.js

```diff
@@ -1310,6 +1335,12 @@ MdPanelRef.prototype._configureTrapFocus = function() {
     this._topFocusTrap.addEventListener('focus', focusHandler);
     this._bottomFocusTrap.addEventListener('focus', focusHandler);
 
+    // Queue remove listeners function
+    this._removeListeners.push(function () {
+      this._topFocusTrap.removeEventListener('focus', focusHandler);
+      this._bottomFocusTrap.removeEventListener('focus', focusHandler);
+    }.bind(this));
```

Comment:
`function.bind( )` used to fail in PhantomJS. Assuming it works with PhantomJS2 ?